### ISSUE TITLE
Jazzy doesn’t run on XC 8 GM when Swift version explicitly specified

### ISSUE THREAD

**Issue by pcantrell** [realm/jazzy#656]
I get:

```
xcrun: error: unable to find utility "xcrun", not a developer tool or in PATH
```

It appears to be this bug at play: https://github.com/segiddins/xcinvoke/issues/6

**Comment by jpsim** [comment]
I've been avoiding this by not specifying a `--swift-version`, but rather using SourceKitten's built in toolchain resolution instead.

**Comment by pcantrell** [comment]
Confirmed, removing `swift_version` fixes it.

**Comment by istx25** [comment]
>Confirmed, removing swift_version fixes it.

Closing.

**Comment by jpsim** [comment]
@istx25 not specifying `swift_version` is a workaround, not a solution.

**Comment by pcantrell** [comment]
https://github.com/segiddins/xcinvoke/pull/7 may fix it, but hasn’t been merged yet — I think because @segiddins wanted somebody to test it on XC 5…?

**Comment by svdo** [comment]
Now CocoaDocs is broken because if this :( CocoaPods automatically adds swift version to podspec, so it's not possible to "not specify swift version" :( Hope the PR on xcinvoke gets merged quickly. If not, we should consider switching to a fork of xcinvoke (or remove it entirely??).

**Comment by segiddins** [comment]
@svdo or you can help verify the xcinvoke PR...

**Comment by svdo** [comment]
Xcode 5 doesn't run on my computer :(

**Comment by pcantrell** [comment]
Sam just cut a new xcinvoke release (0.3.0) that may fix this.

**Comment by jpsim** [comment]
Fixed in #691.
